ui/src/page/start_game.rs

```diff
@@ -16,20 +15,22 @@ pub struct Model {
     game_name_field: String,
     player_name_field: String,
     session: Session,
+    request_state: RequestState,
+}
+
+enum RequestState {
+    Ready,
+    Waiting,
+    Failed(String),
```

Comment:
"Ready" as in ready to start a new game,  "Waiting", as in waiting to hear back, and "Failed" as in something went wrong when trying to start the game.